src/transport/SecureSessionMgr.cpp

```diff
@@ -110,55 +124,94 @@ CHIP_ERROR SecureSessionMgr::SendMessage(PayloadHeader & payloadHeader, NodeId p
     mPeerConnections.MarkConnectionActive(state);
 
     {
-        uint8_t * data = nullptr;
+        uint8_t * data         = nullptr;
+        uint8_t * p            = nullptr;
+        uint32_t msgId         = 0;
+        uint32_t payloadLength = 0;
+        uint16_t len           = 0;
         PacketHeader packetHeader;
         MessageAuthenticationCode mac;
 
         const uint16_t headerSize = payloadHeader.EncodeSizeBytes();
         uint16_t actualEncodedHeaderSize;
         uint16_t totalLen = 0;
         uint16_t taglen   = 0;
-        uint32_t payloadLength; // Make sure it's big enough to add two 16-bit
-                                // ints without overflowing.
```

Comment:
This comment got lost, and is actually somewhat important.  Please put it back.

Reply:
Acked